packages/orbit-components/src/hooks/useMediaQuery/index.js

```diff
@@ -1,10 +1,89 @@
 // @flow
```

Comment:
I moved the logic from `QueryContextProvider` here because `useMediaQuery` needs to work both with and without it. And this is a better place for it anyway.